test/libsolidity/semanticTests/storageLayoutSpecifier/variable_cleanup_sstore.sol

```diff
@@ -0,0 +1,16 @@
+contract C {
```

Comment:
Just noticed that this and `variable_cleanup.sol` are missing the layout specifier, which is the whole point of these tests.

Reply:
Fixed.